docs/tasks/debug-application-cluster/logging-stackdriver.md

```diff
@@ -146,3 +146,98 @@ or to [BigQuery](https://cloud.google.com/bigquery/) to run further
 analysis. Stackdriver Logging offers the concept of sinks, where you can
 specify the destination of log entries. More information is available on
 the Stackdriver [Exporting Logs page](https://cloud.google.com/logging/docs/export/configure_export_v2).
+
+## Configuring Stackdriver Logging Agents
+
+Sometimes the default installation of Stackdriver Logging may not suite your needs, for example:
+
+* You may want to add more resources because default performance doesn't suite your needs
+* You may want to introduce additional parsing to extract more metadata from you log messages,
+like severity or source code reference
+* You may want to send logs not only to Stackdriver or send it to Stackdriver only partially
+
+In this case you need to be able to change the parameters of `DaemonSet` and `ConfigMap`.
+
+### Prerequisites
+
+If you're on GKE and Stackdriver Logging is enabled in your cluster, you cannot change its
+parameters. Likewise, if you're not on GKE, but Stackdriver Logging is installed as an addon,
+you won't be able to change deployment parameters using Kubernetes API. To make it possible
+to change parameters of Stackdriver Logging agents, you should switch to the API object
+deployment, when Stackdriver Logging is installed into a running cluster that didn't have any
+cluster logging solutions installed before that.
+
+You can find notes on how to install Stackdriver Logging agents into a running cluster in the
+[Deploying section](#deploying).
```

Comment:
Deploying section is missing because I was doing two PRs in parallel: this and https://github.com/kubernetes/kubernetes.github.io/pull/2922 Now that the latter is merged, link should be working.

> Also does it make sense to link to/from cluster/addons/fluentd-gcp somewhere in this page?

Good question. I deliberately didn't mention it and made a local copy of the `DaemonSet` and the `ConfigMap` to avoid problems with the k8s versions, i.e. if something changes in master, docs should not be affected. Does it make sense?